src/Microsoft.Health.Dicom.Api/Web/DicomStreamContent.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.Health.Dicom.Api.Web;
 
-public class DicomStreamContent
+public class DicomStreamContent : IHttpStreamContent
```

Comment:
nit: we can probably call this `StreamContent`?

Reply:
Or maybe something related to the fact is part of a multi-part?

Reply:
It is http stream content with data as stream and headers as keyvalue pair